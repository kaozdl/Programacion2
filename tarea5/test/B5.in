# B
ingresar_localidades (15,ulsan) .
iniciar_pedidos 5
ingresar_pedidos ulsan (1000,chapas) 5 ulsan (2000,comestibles) 3 .
ingresar_pedidos ulsan (1000,medicamentos) 5 ulsan (2000,medicamentos) 3 .
ingresar_pedidos ulsan (1000,chapas) 1 ulsan (2000,comestibles) 4 .
imprimir_totales
procesar_rango 2
procesar_rango 1
procesar_rango 1
Fin
