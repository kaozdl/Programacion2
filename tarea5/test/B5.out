1>2>3>
 IDENTIFICADORES DE LOCALIDADES 
(1,ulsan)
4>5>6>7>INFORME POR RANGOS
9000
0
0
0
0
8>9>ulsan
(1000,chapas)(2000,comestibles)(2000,medicamentos)(2000,comestibles)(1000,chapas)(1000,medicamentos)
10>ulsan

11>Fin.
